core/backend/src/NativeHost.ts

```diff
@@ -96,7 +96,7 @@ class NativeAppHandler extends IpcHandler implements NativeAppFunctions {
     NativeHost.overrideInternetConnectivity(by, status);
   }
   public async getConfig(): Promise<any> {

```

Comment:
do we really want to keep this method? I see you removed the (only?) call to it. I think we should remove it.

What did `getContainer` do? At the very least it should be declared to return `string[]`

Reply:
Good point, I didn't catch that it was the only reference to it. I'll remove it.

`getContainer` just sent a deep copy of the "container" of variables in the `Config` class.

Reply:
This change has been made.

Reply:
@kabentley mind taking a quick review here with all the updates?